src/routes/CardAdd/index.tsx

```diff
@@ -30,15 +33,30 @@ const CardAdd = () => {
   const cardSecurityCode = useCardNumberData(SECURITY_CODE_LENGTH_MAX)
   const cardPassword = useCardNumberData(PASSWORD_LENGTH_MAX)
 
-  const validationNavigate = () => {
+
+  const isValidationCardData = () => {
     const design = cardDesign.validation
     const number = !cardNumber.validation.includes(false)
     const expiration = !cardExpirationDate.validation.includes(false)
     const owner = ownerName.validation
     const security = cardSecurityCode.validation[0]
     const password = cardPassword.validation[0] && cardPassword.validation[1]
+    
+    return design && number && expiration && owner && security && password
+  }
 
-    if (design && number && expiration && owner && security && password) {
+  const onSubmitNavigator = (e: React.MouseEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    if (isValidationCardData()) {
+      setCardItem({
+        nickName: '',
+        cardDesign: cardDesign.cardDesign,
+        cardNumber: cardNumber.cardNumberData,
+        cardExpirationDate: cardExpirationDate.cardExpirationDate,
+        ownerName: ownerName.ownerName,
+        cardSecurityCode: cardSecurityCode.cardNumberData.num1,
+        cardPassword: { password1: cardPassword.cardNumberData.num1, password2: cardPassword.cardNumberData.num2 },
+      })
       navigate('/card-add-complete')
     } else {
       alert('입력한 값을 확인해주세요.')
```

Comment:
`else`문을 위로 올려서 ealry return을 활용해주시면 코드 보기가 한결 편해질거 같네요!
```suggestion
  const onSubmitNavigator = (e: React.MouseEvent<HTMLFormElement>) => {
    e.preventDefault()

    if (!isValidationCardData()) {
      alert('입력한 값을 확인해주세요.')
      return;
    })

      setCardItem({
        nickName: '',
        cardDesign: cardDesign.cardDesign,
        cardNumber: cardNumber.cardNumberData,
        cardExpirationDate: cardExpirationDate.cardExpirationDate,
        ownerName: ownerName.ownerName,
        cardSecurityCode: cardSecurityCode.cardNumberData.num1,
        cardPassword: { password1: cardPassword.cardNumberData.num1, password2: 
        cardPassword.cardNumberData.num2 
},
```